app/models/__init__.py

```diff
@@ -39,6 +39,7 @@
 ADDRESS_KEY = "address"
 AGGREGATE_KEY = "aggregate"
 ARCHITECTURE_KEY = "arch"
+ATTACHMENTS_KEY = 'attachments'
```

Comment:
BTW I think we never hit this issue before because there aren't any KernelCI tests using "attachments" at the moment.

Reply:
What is more, I remember fixing this issue some time ago, so it's both unused and a regression.